OpenRA.Game/FileSystem/ZipFile.cs

```diff
@@ -236,14 +231,13 @@ public static bool TryParseReadWritePackage(string filename, out IReadWritePacka
 				return false;
 			}
 
-			var s = new FileStream(filename, FileMode.Open);
-			package = new ReadWriteZipFile(s, filename);
+			package = new ReadWriteZipFile(filename);
 			return true;
 		}
 
 		public static IReadWritePackage Create(string filename)
 		{
-			return new ReadWriteZipFile(null, filename);
+			return new ReadWriteZipFile(filename);
```

Comment:
Is filename always not yet existing file? Before it was created here empty MemoryStream, now it will load the file if it exists.